src/configuration/ConfigEditor.tsx

```diff
@@ -31,14 +31,14 @@ import { HelpfulLinks } from "./HelpfulLinks";
 import { LimitsSettings } from "./LimitsSettings";
 import { PromSettings } from './PromSettings';
 
-export enum DataSourceType {
-  Alertmanager = 'alertmanager',
-}
+// export enum DataSourceType {
+//   Alertmanager = 'alertmanager',
+// }
 
 export type Props = DataSourcePluginOptionsEditorProps<PromOptions>;
 export const ConfigEditor = (props: Props) => {
   const { options, onOptionsChange } = props;
-  const alertmanagers = Object.values(config.datasources).filter((ds) => ds.type === DataSourceType.Alertmanager);
+  // const alertmanagers = Object.values(config.datasources).filter((ds) => ds.type === DataSourceType.Alertmanager);
```

Comment:
if this is unused we can remove this comment

Reply:
removed